operations/companies.md

```diff
@@ -244,36 +326,196 @@ Adds a new company. Note this operation supports [Portfolio Access Tokens](../gu
+      "Department": "Sales",
+      "DunsNumber": "123456789",
+      "CreditRating": {
+        "Basic": "PaymentRequiredUpfront"
+      },
+      "ReferenceIdentifier": "da34b396-77e3-495a-bd61-aecf00a3f19d",
+      "WebsiteUrl": "https://www.mewssystems.com",
+      "ExternalIdentifier": "company0001"
+    },
+    {
+      "Id": "da34b396-41f7-47f6-8847-aecf00a3f19e",
+      "ChainId": "5fcd1933-22f2-40b9-84da-7db04cbecec2",
+      "Name": "Sample Company 2",
+      "IsActive": true,
+      "Number": 12,
+      "Identifier": null,
+      "TaxIdentifier": null,
+      "AdditionalTaxIdentifier": null,
+      "ElectronicInvoiceIdentifier": null,
+      "InvoiceDueInterval": "P2DT23H",
+      "AccountingCode": null,
+      "MotherCompanyId": null,
+      "CreatedUtc": "2022-07-11T09:56:54Z",
+      "UpdatedUtc": "2022-07-11T09:56:54Z",
+      "AddressId": null,
+      "BillingCode": null,
+      "Iata": "PAO",
+      "Telephone": "111-222-333",
+      "InvoicingEmail": "sample2@mews.com",
+      "ContacPerson": "Sample Person 2",
+      "Contact": "Contact Info 2",
+      "Notes": "Note 2",
+      "Options": {
+        "Invoiceable": true,
+        "AddFeesToInvoices": false
+      },
+      "Department": "Accounting",
+      "DunsNumber": "987654321",
+      "CreditRating": {
+        "Basic": "CreditOk"
+      },
+      "ReferenceIdentifier": "a58ff7cb-77e3-495a-bd61-aecf00a3f19d",
+      "WebsiteUrl": "https://www.mews.com",
+      "ExternalIdentifier": "company0002"
+    }
+  ],
+  "Cursor": "da34b396-41f7-47f6-8847-aecf00a3f19e"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Companies` | array of [Company](companies.md#company) | required | The company profiles of the enterprise. |
+| `Cursor` | string | optional | Unique identifier of the last and hence oldest company item returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older companies. If [Limitation](../guidelines/pagination.md#limitation) is specified in the request message, then Cursor will always be included in the response message; this is true even when using Extents set to false so that no actual data is returned. |
+
+#### Company
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the company. |
+| `ChainId` | string | required | Unique identifier of the chain. |
+| `Name` | string | required | Name of the company. |
+| `MotherCompanyId` | string | optional | Unique identifier of mother company. |
+| `InvoicingEmail` | string | optional | Email for issuing invoices to the company. |
+| `WebsiteUrl` | string | optional | The website url of the company. |
+| `InvoiceDueInterval` | string | optional | The maximum time, when the invoice has to be be paid in ISO 8601 duration format. |
+| `Options` | [CompanyOptions](companies.md#companyoptions) | optional | Options of the company. |
```

Comment:
❓ Current documentaion has `Options` as `required`. What is right and what is left? 🤔

Reply:
Ah, yes, since it's an object and we always serialize it, it should be `required`. Also, you're missing a title on that type.